include/openvswitch/ovs-p4rt.h

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Defines the public interface to an externally-supplied module
+ * that permits OvS to communicate with the P4 control plane.
+ */
+
+#ifndef OPENVSWITCH_OVS_P4RT_H
+#define OPENVSWITCH_OVS_P4RT_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#include <stdint.h>
```

Comment:
This would normally be placed at the top of the file, between the #include guard and the start of the extern "C" block.